dbos/_workflow_commands.py

```diff
@@ -55,24 +53,52 @@ def list_workflows(
         if status is not None:
             input.status = cast(WorkflowStatuses, status)
         input.application_version = appversion
-        input.limit = li
+        input.limit = limit
 
         output: GetWorkflowsOutput = sys_db.get_workflows(input)
-
         infos: List[WorkflowInformation] = []
+        for workflow_id in output.workflow_uuids:
+            info = _get_workflow_info(
+                sys_db, workflow_id, request
+            )  # Call the method for each ID
+            if info is not None:
+                infos.append(info)
```

Comment:
`get_workflows` should return more information imho, rather than us having to do more queries. This comes up in other parts of the system.